Library/Homebrew/utils.rb

```diff
@@ -663,7 +663,7 @@ def build_query_string(query, qualifiers)
 
   def build_search_qualifier_string(qualifiers)
     {
-      :repo => "Homebrew/homebrew",
+      :repo => "Homebrew/brew",
```

Comment:
If I understand correctly, this is invoked on a failed search for a formula (at least judging from the referenced issue), so searching in `Homebrew/homebrew-core` seems to make more sense. `Homebrew/brew` is not going to yield any sensible results for searching involving formula names, right?